docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -193,13 +193,15 @@ quarkus.http.ssl.certificate.trust-store-password=the_trust_store_secret
 quarkus.http.ssl.client-auth=required                                      <3>
 quarkus.http.auth.permission.default.paths=/*                              <4>
 quarkus.http.auth.permission.default.policy=authenticated
+quarkus.http.insecure-requests=disabled                                    <5>
```

Comment:
@cescoffier In this context it makes no difference because Quarkus Security enforces it with its own constraints, the `authenticated` policy will fail.

Reply:
Are you sure using `quarkus.http.insecure-requests=disabled ` isn't more effective than HTTP security policy when it comes to disabling HTTP protocol? I think it means Vert.x will not pass any insecure requests to route handlers so there will be way less of processing. The `quarkus.http.insecure-requests=disabled ` configuration property should have positive impact.

Reply:
@michalvavrik May be it is is but in this example it is unnecessary, where Quarkus security is already configured.

I'd consider documenting it here as an alternative, IMHO that would better IMHO

Reply:
Provided app use `io.quarkus.security.runtime.X509IdentityProvider` and don't replace it with something more complex, I think it results in a same thing, yeah. If it's documented separately with context user will have all information.